pom.xml

```diff
@@ -43,25 +43,42 @@
   		<plugin>
   			<groupId>org.apache.maven.plugins</groupId>
   			<artifactId>maven-compiler-plugin</artifactId>
-  			<version>2.3.1</version>
+  			<version>3.8.1</version>
   			<configuration>
-  			  <source>1.6</source>
-  			  <target>1.6</target>
+  			  <source>1.8</source>
+  			  <target>1.8</target>
   			</configuration>
   		</plugin>
-                <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-source-plugin</artifactId>
-                        <version>2.2.1</version>
-                        <executions>
-                          <execution>
-                            <id>attach-sources</id>
-                            <goals>
-                              <goal>jar</goal>
-                            </goals>
-                          </execution>
-                        </executions>
-                </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-source-plugin</artifactId>
+            <version>2.2.1</version>
+            <executions>
+              <execution>
+                <id>attach-sources</id>
+                <goals>
+                  <goal>jar</goal>
+                </goals>
+              </execution>
+            </executions>
+        </plugin>
+        <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+            <annotationProcessorPaths>
+              <path>
+                <groupId>com.google.auto.service</groupId>
+                <artifactId>auto-service</artifactId>
+                <version>1.0-rc7</version>
+              </path>
```

Comment:
There was a bug in IntelliJ where the project wasn't building properly without adding the auto-service dependency there. However, since simply running native maven appears to build and test properly, I will remove it.